reference/fleet/migrate-from-beats-to-elastic-agent.md

```diff
@@ -1,7 +1,11 @@
 ---
 mapped_pages:
```

Comment:
Should add a serverless: unavailable badge to the heading titled "[Set up Fleet Server (self-managed deployments only)](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/3238/reference/fleet/migrate-from-beats-to-elastic-agent#_set_up_fleet_server_self_managed_deployments_only)" on this page